src/context/WishListContext.jsx

```diff
@@ -0,0 +1,16 @@
+import { createContext, useContext, useState } from "react";
+
+const WishListContext = createContext();
```

Comment:
Its better practice to provide default values in the context for any failture.

Reply:
okay thanks for the suggestion added